README.md

```diff
@@ -243,7 +243,7 @@ In the generic usage, you can chain `search` methods:
 ```ruby
 doc.book.search("[data-type='page']").search("figure")
 ```
-
+<!-- There is no other mention above of `my_chapter` was this intentional? -->
```

Comment:
Made this comment as a note to myself to ask this question. Don't plan on pushing it